packages/core/src/js/feedback/FeedbackWidget.theme.ts

```diff
@@ -0,0 +1,41 @@
+import { Appearance } from 'react-native';
+
+import { getColorScheme, getFeedbackDarkTheme, getFeedbackLightTheme } from './integration';
+
+/**
+ * Get the theme for the feedback widget based on the current color scheme
+ */
+export function getTheme(): FeedbackWidgetTheme {
+  const userTheme = getColorScheme();
+  const colorScheme = userTheme === 'system' ? Appearance.getColorScheme() : userTheme;
+  const lightTheme = { ...LightTheme, ...getFeedbackLightTheme() };
+  const darkTheme = { ...DarkTheme, ...getFeedbackDarkTheme() };
+  return colorScheme === 'dark' ? darkTheme : lightTheme;
+}
+
+export interface FeedbackWidgetTheme {
+  accent: string;
```

Comment:
I think we are missing some properties compared to JS.

https://github.com/getsentry/sentry-javascript/blob/a3e44f222caef518ea1f006dc75c1a0c98b2617e/packages/core/src/types-hoist/feedback/theme.ts#L13-L31

Reply:
Good catch 👍 

I tried to use just one accent for simplicity but this resulted in the [issue reported above](https://github.com/getsentry/sentry-react-native/pull/4677#discussion_r2014646665).

> The rest is probably not applicable to the mobile version, right?

I think too the rest do not apply to the mobile version.

Updated with c831eeeac0886bed343c0755c7d8bed37cf8d984